include/zephyr/xen/public/io/libxenvchan.h

```diff
@@ -0,0 +1,101 @@
+/**
```

Comment:
Zephyr license style requires single line SPDX Identifier at the start of the file:
`/* SPDX-License-Identifier: MIT */`

To find an example, please refer to other public headers in mainline.

Reply:
done